pydax/loaders/_table.py

```diff
@@ -55,9 +56,15 @@ def load(self, path: Union[_typing.PathLike, Dict[str, str]], options: SchemaDic
             else:
                 dtypes[column] = type_
 
+        names = None
```

Comment:
Does `names` default to `None` in `read_csv`? I don't see this in the document. Perhaps it's better if we simply do not provide `names` if `header` is False?

Reply:
`names` is `None` by default. I thought about doing that but I wanted to avoid having 2 versions of `read_csv`